src/test/java/uk/gov/pay/ledger/queue/EventMessageHandlerTest.java

```diff
@@ -31,6 +32,9 @@
     @Mock
     private TransactionService transactionService;
 
+    @Mock
+    private TransactionMetadataService transactionMetadataService;
```

Comment:
This is not used

Reply:
Neither is TransactionService, but without either mocks the test fails:
`Wanted but not invoked:
eventQueue.markMessageAsProcessed(
    <any uk.gov.pay.ledger.queue.EventMessage>
);`

Reply:
Neither is TransactionService, but without either mocks the test fails:
Wanted but not invoked: eventQueue.markMessageAsProcessed( <any uk.gov.pay.ledger.queue.EventMessage> );

Reply:
👍 didn't notice `@InjectMocks`